### PR TITLE
Basic wave implementation

### PR DESCRIPTION
**Pull request by hej2010**

### CODE REVIEW

**Review by EmElw — CHANGES_REQUESTED** [review]
Delegate wave-functionality to a separate class

File: core/src/com/pottda/game/model/Character.java [hunk 1]
```diff
@@ -20,10 +21,11 @@
     /**
      * Current health of a character
      */
-    public int currentHealth;
+    int currentHealth;
     private static Map<Stat, Double> stats;
     private Vector2f movementVector;
 
+    public static Character player;
```
Comment (EmElw) [inline core/src/com/pottda/game/model/Character.java:28]: Probably fine for now, but we should try to find a better way of doing this.

File: core/src/com/pottda/game/PoTDA.java [hunk 2]
```diff
@@ -58,13 +60,20 @@
...
         MAIN_MENU,
         MAIN_CHOOSE,
         MAIN_CONTROLS
     }
 
+    private int currentLevel = 0;
+    private int currentWave = 0;
+    private int nrOfWaves = 0;
+    private long startTime = 0;
+    private static final int waitingTimeSeconds = 5;
```
Comment (EmElw) [inline core/src/com/pottda/game/PoTDA.java]: Should be called WAITING_TIME_SECONDS ;)

**Review by LatteG — CHANGES_REQUESTED** [review]
Change it so DumbAIController uses Character.player instead of goal

**Review by EmElw — APPROVED** [review]

**Review by LatteG — APPROVED** [review]

**Pull request merged**